src/util/map.js

```diff
@@ -77,3 +77,19 @@ export const drillUpDown = (layerConfig, parentId, parentGraph, level) => ({
     ],
     parentGraphMap: {},
 });
+
+// Called when plugin maps enter or exit fullscreen
+export const onFullscreenChange = (map, isFullscreen = false) => {
+    map.resize();
+
+    if (!isFullscreen) {
+        const bounds = map.getLayersBounds();
+
+        if (Array.isArray(bounds)) {
```

Comment:
Does it make a difference if the array is empty?

Reply:
We have a check for this in `maps-gl`, so should not be needed here:
https://github.com/dhis2/maps-gl/blob/master/src/utils/geometry.js#L18